tests/conftest.py

```diff
@@ -19,3 +19,19 @@ def teardown():
     request.addfinalizer(teardown)
 
     return flaskr.app
+
+
+@pytest.fixture(scope='function')
+def base_url(live_server):
+    return live_server.url()
+
+
+@pytest.fixture(scope='function')
+def sensitive_url(request):
+    return False
+
+
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Each test has it's own instance of the application, so we need to override the session scoped `_verify_url` fixture with a function scoped one.